e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts

```diff
@@ -0,0 +1,41 @@
+describe('chef datafeed', () => {
+const now  = Cypress.moment().format('MMDDYYhhmmss');
```

Comment:
also, please take care of the indentation 😁

Reply:
Done